### DISCUSSION TITLE
Hoymiles 1500 drosseln über OpenDTU NordicSkills

### DISCUSSION THREAD

**Discussion by hmayer61** [tbnobody/OpenDTU#1962] [Q&A]
Hallo,  kann mir jemand helfen, wie ich einen Hoymiles 1500 über die openDTU von NordicSkills auf 800 Watt Ausgangsleistung drosseln kann ?    Lt. opendtu.solar  doku sollte das gehen wenn ich den Eintrag richtig interpretiere. 

- Aktuelle Wechselrichtergrenze anzeigen und einstellen 

Leider kann ich nicht wirklich finden wo und wie das gehen soll. 

Vielen Dank.

**Comment by ghost** [comment]
Hi,

wenn die Firmware nicht zu alt ist geht das hier:
(Klicke auf das stilisierte Zeigerinstrument)

![1](https://github.com/tbnobody/OpenDTU/assets/155244148/3f1016e1-d145-453c-8e46-621fe7080857)

Dann erscheint Folgendes:

![2](https://github.com/tbnobody/OpenDTU/assets/155244148/d7416229-3c8f-4067-a08d-07264a2e7421)

**Reply by hmayer61** [reply]
oh mann,   wie blind kann man sein..   :-)   vielen Dank fürs guiden.
